### ISSUE TITLE
[question] update frequency

### ISSUE THREAD

**Issue by jathek** [fyr77/EnvyUpdate#47]
When minimized to the taskbar, does EnvyUpdate check once in a while? Perhaps once a day or something? Or does it only check once, when it is first opened? I checked the readme but I didn't find anything about the timing.

**Comment by fyr77** [comment]
Good point, I should really add this info somewhere. It is set to update once every 4 hours, but I could absolutely allow users to set this. It also checks at startup and after driver installation.

**Comment by jathek** [comment]
Yes, once a day would be enough for me, but it's good to know that it does check sporadically in the background, I wasn't sure.
